sources/europe/ch/Stadt_Bern_10cm-2020-WMS.geojson

```diff
@@ -0,0 +1,55 @@
+{
+    "type": "Feature",
+    "properties": {
+        "license_url": "https://www.bern.ch/themen/planen-und-bauen/geodaten-und-plane/nutzungsbedingungengeodaten/nutzungsbedingungen",
+        "privacy_policy_url": "https://www.bern.ch/disclaimer",
+        "start_date": "2020",
+        "end_date": "2020",
+        "id": "Bern-bern2020-wms",
+        "attribution": {
+            "url": "https://www.bern.ch/themen/planen-und-bauen/geodaten-und-plane/nutzungsbedingungengeodaten/nutzungsbedingungen",
+            "text": "Geodaten Stadt Bern",
+            "required": true
+        },
+        "name": "Stadt Bern 10cm (2020)",
+        "available_projections": [
+            "CRS:84",
+            "EPSG:2056",
+            "EPSG:21781",
+            "EPSG:4326"
+        ],
+        "url": "https://map.bern.ch/wms/OpenData/proxy.php?LAYERS=luftbild_2020&STYLES=default&FORMAT=image/jpeg&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
+        "country_code": "CH",
+        "type": "wms",
+        "category": "photo",
+        "best": true,
+        "min_zoom": 10
```

Comment:
I'm not sure about the usefulness of min_zoom for wms sources, as zoom levels are something that do not exist for WMS but tms/wmts. I just used the value 10 from Stadt_Bern_10cm-2016-WMS.geojson